src/js/personalization/dashboard-beta/containers/DashboardApp.jsx

```diff
@@ -13,6 +14,18 @@ import BetaApp, { features } from '../../beta-enrollment/containers/BetaApp';
 import RequiredLoginView from '../../../common/components/RequiredLoginView';
 import DowntimeNotification, { services } from '../../../../platform/monitoring/DowntimeNotification';
 
+import profileManifest from '../../va-profile-beta/manifest.json';
+import accountManifest from '../../account-beta/manifest.json';
+
+const scroller = Scroll.animateScroll;
+const scrollToTop = () => {
+  scroller.scrollTo(0, {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
```

Comment:
Do multiple apps do this? Seems like something we could convert to a general util at some point.

Reply:
Agree... not all apps do this, but many do. This is a common UX problem for us because of how browsers handle content being loaded in the middle of the page